.github/workflows/build.yml

```
@@ -77,50 +77,19 @@ jobs:
         docker push ${{env.DOCKER_REPO}}:${{env.BRANCH_TAG}}
         docker push ${{env.DOCKER_REPO}}:${{env.SHA_TAG}}
 
-  #master
-  deploy_to_paas_QA:
 
-    name: Deploy to Paas
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
-    needs: build
-    env:
-      SHA_TAG: ${{needs.build.outputs.sha_tag}}
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Pin Terraform version
-      uses: hashicorp/setup-terraform@v1.2.1
+    - name: Trigger QA Deployment
+      if: ${{ success() }}
```

```suggestion
      if: ${{ success() && `github.ref == 'refs/heads/master' }}
```
add condition to only deploy if `github.ref == 'refs/heads/master'`

---

done..